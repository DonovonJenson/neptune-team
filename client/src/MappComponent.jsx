import React from 'react';

var MappComponent = (props) => (
  <div className="row">
    <div id="mapblock_content">  
      <img style={{maxWidth: "100%", minWidth: "100%", height: "375px"}} src="https://static1.squarespace.com/static/4f6792f624ac778428aca39d/4f760d4b6a9b61f6cc2748c3/51a798f6e4b01ba7ba19b03b/1387223684177/google-maps-squarespace-birds-eye-view-01.jpg?format=2500w"/>
      <p className="mapblock_content">Any Map Notes Go Here</p>
      <div className="input-group col-md-12">
        <input type="text" className="form-control"></input>
        <span className="input-group-btn">
          <button className="btn btn-primary" type="button">Search This Location</button>
        </span>
      </div>
    </div>
  </div>
  )



export default MappComponent;
