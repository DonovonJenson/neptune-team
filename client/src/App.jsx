import React from 'react';
import MappComponent from './MappComponent.jsx';
import PhotoInfo from './PhotoInfo.jsx';
import TipsBlock from './TipsBlock.jsx';
import TopTweetsInfo from './TopTweetsInfo.jsx';
import WalkabilityInfo from './WalkabilityInfo.jsx';
import TwitterTrends from './TwitterTrends.jsx';


class App extends React.Component {
  constructor() {
    super();
    this.state = {};
  }

  render() {
    return (    
        <div className="container-fluid">
          <div className="row">
            <div className="col-12-sm" id="titlebar">
              <h1 id="titlefont">InfoMapp</h1>
              <h2 id="subtitlefont">Learn More With Just A Click</h2>
            </div>
          </div>
          <div className="row">
            <div className="col-sm-6">  
              <div id="mapblock" className="vertical-center">         
                <MappComponent />
              </div>
              <div id="twittertrends">
                <TwitterTrends />
              </div>
            </div>
            <div className="col-sm-6">
              <div id="infoblock">
                <div className="row">
                  <div className="col-sm-6">
                    <div id="walkabilityblock">
                      <WalkabilityInfo />
                    </div>
                  </div>

                  <div className="col-sm-6">
                    <div id ="tipsblock">
                      <TipsBlock />
                    </div>
                  </div>





                </div>
                <div id="toptweetsblock">
                  <TopTweetsInfo />
                </div>
                <div id="photoblock">
                  <PhotoInfo />
                </div>
              </div>
            </div>
            
          </div>
        </div>
    );
  }
}

export default App;





