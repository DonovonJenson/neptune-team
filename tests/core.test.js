const modules = require('../client/src/App.jsx');
import React from 'react';
import ReactDOM from 'react-dom';
import App from '../client/src/App.jsx';
import MappComponent from '../client/src/MappComponent.jsx';
import renderer from 'react-test-renderer';
import { shallow, mount, render, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
});

// note: use 'u' to update upon render with acceptable changes
it('renders a snapshot of App Component1', () => {
  const tree = renderer.create(<App/>).toJSON();
  expect(tree).toMatchSnapshot();
});

describe('<App />', () => {
  it('renders 1 app component', () => {
    const component = shallow(<App />);
    expect(component).toHaveLength(1);
  })
})

