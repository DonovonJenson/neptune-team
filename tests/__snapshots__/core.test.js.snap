// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of App Component1 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col-12-sm"
      id="titlebar"
    >
      <h1
        id="titlefont"
      >
        InfoMapp
      </h1>
      <h2
        id="subtitlefont"
      >
        Learn More With Just A Click
      </h2>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-sm-6"
    >
      <div
        className="vertical-center"
        id="mapblock"
      >
        <div
          className="row"
        >
          <div
            id="mapblock_content"
          >
            <img
              src="https://static1.squarespace.com/static/4f6792f624ac778428aca39d/4f760d4b6a9b61f6cc2748c3/51a798f6e4b01ba7ba19b03b/1387223684177/google-maps-squarespace-birds-eye-view-01.jpg?format=2500w"
              style={
                Object {
                  "height": "375px",
                  "maxWidth": "100%",
                  "minWidth": "100%",
                }
              }
            />
            <p
              className="mapblock_content"
            >
              Any Map Notes Go Here
            </p>
            <div
              className="input-group col-md-12"
            >
              <input
                className="form-control"
                type="text"
              />
              <span
                className="input-group-btn"
              >
                <button
                  className="btn btn-primary"
                  type="button"
                >
                  Search This Location
                </button>
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        id="twittertrends"
      >
        <div>
          <p
            className="twittertrends_content"
          >
            Twitter Trends Go Here!
          </p>
        </div>
      </div>
    </div>
    <div
      className="col-sm-6"
    >
      <div
        id="infoblock"
      >
        <div
          className="row"
        >
          <div
            className="col-sm-6"
          >
            <div
              id="walkabilityblock"
            >
              <p
                className="walkabilityblock_content"
              >
                Walkability Info Goes Here
              </p>
            </div>
          </div>
          <div
            className="col-sm-6"
          >
            <div
              id="tipsblock"
            >
              <div>
                <p
                  className="tipsblock_content"
                >
                  Tips From Locals Go Here
                </p>
                <p>
                  Running with the bulls in Bolton, OH  is amazing.
                </p>
                <p>
                  -Mel Melverson
                </p>
                <br />
                <p>
                  I found the library to be both quaint and huge at the same time.
                </p>
                <p>
                  -Tony T.
                </p>
                <br />
                <p>
                  How much wood would a woodchuck chuck in Bolton? Visist and find out!
                </p>
                <p>
                  Susan Solomon
                </p>
                <br />
                <p>
                  Running with the bulls in Bolton, OH  is amazing.
                </p>
                <p>
                  -Mel Melverson
                </p>
                <br />
                <p>
                  I found the library to be both quaint and huge at the same time.
                </p>
                <p>
                  -Tony T.
                </p>
                <br />
                <p>
                  How much wood would a woodchuck chuck in Bolton? Visist and find out!
                </p>
                <p>
                  Susan Solomon
                </p>
                <p>
                  Running with the bulls in Bolton, OH  is amazing.
                </p>
                <p>
                  -Mel Melverson
                </p>
                <br />
                <p>
                  I found the library to be both quaint and huge at the same time.
                </p>
                <p>
                  -Tony T.
                </p>
                <br />
                <p>
                  How much wood would a woodchuck chuck in Bolton? Visist and find out!
                </p>
                <p>
                  Susan Solomon
                </p>
                <br />
                <p>
                  I found the library to be both quaint and huge at the same time.
                </p>
                <p>
                  -Tony T.
                </p>
                <br />
                <p>
                  How much wood would a woodchuck chuck in Bolton? Visist and find out!
                </p>
                <p>
                  Susan Solomon
                </p>
                <p>
                  sdvdfvfd
                </p>
                <p>
                  sdvgbbgbfbfgbfgbfgbfgbg
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          id="toptweetsblock"
        >
          <div>
            <p
              className="toptweetsblock_content"
            >
              Top 3 Tweets FROM A Location
            </p>
            <p
              className="toptweetsblock_content"
            >
              Top 3 Tweets ABOUT A Location
            </p>
          </div>
        </div>
        <div
          id="photoblock"
        >
          <p
            className="photoblock_content"
          >
            Photo goes Here
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
